sdk/cognitivelanguage/azure-ai-language-conversations/swagger/README.md

```diff
@@ -71,6 +70,26 @@ Customizations that should eventually be added to central autorest configuration
 
 ### General customizations
 
+#### Add rest api docs to operations
+
+```yaml
+directive:
+- from: analyzeconversations-authoring.json
+  where: $.paths.*.*
+  transform: |
+    var operationId = $.operationId.replace(/_/g, "/").replace(/([a-z0-9])([A-Z])/g, "$1-$2").toLowerCase();
+    $.description = $.description + " See https://learn.microsoft.com/rest/api/language/" + operationId + " for more information.";
```

Comment:
If possible, would you want newline characters here e.g., "\n"?

Reply:
Do you mean after the description? The generator adds a newline between this and the param list:

https://github.com/Azure/azure-sdk-for-python/blob/8add6552b1096174e3b1ee8860a1de6723893eee/sdk/cognitivelanguage/azure-ai-language-conversations/azure/ai/language/conversations/_operations/_operations.py#L89-L94

Reply:
Right. Just aesthetically, I was thinking this looks better, and also highlights the link to more info rather than a possibly large paragraph:

```
Analyzes the input conversation utterance. {Plus potentially a lot more text that may detract from the link...}

See https://learn.microsoft.com/rest/api/language/conversation-analysis-runtime/analyze-conversation for more information.

:param task: A single conversational task to execute. Required.
```

Just a nit. This is also how the C# doc comments end up looking (or should, last I saw).

Reply:
Oh I see! I like that separation better, will update.